Doc/library/datetime.rst

```diff
@@ -1501,33 +1624,27 @@ Instance attributes (read-only):
 
    .. versionadded:: 3.6
 
+`time` objects support comparison of :class:`.time` to :class:`.time`,
+where *a* is considered less
+than *b* when *a* precedes *b* in time.  If one comparand is naive and the other
+is aware, :exc:`TypeError` is raised if an order comparison is attempted. For equality
```

Comment:
same here after TypeError

Reply:
Sorry, I am not sure what you mean here.  `TypeError` only seems to have one space after it, could you please clarify?

Reply:
Oops, my bad here sorry!

Reply:
No problem.  Thanks for your review.